auth/template.yaml

```diff
@@ -780,5 +766,5 @@ Outputs:
     Value: !Ref SharedSignalSecretsManager
     
   CognitoSecretArn:
-    Description: The name of the Cognito secret
-    Value: !Ref CognitoClientSecret
+    Description: The name of the Cognito secret. Can be the ARN or name of the secret to retrieve
+    Value: !Sub arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:/cognito/client-secret
```

Comment:
You can't use intrinsic functions in the `Value` property of a param AFAIK

Reply:
Removed it for now - it was only used in testing, falling back to secret name instead of arn